test/hotspot/jtreg/compiler/c2/cr7200264/TestIntVect.java

```diff
@@ -21,622 +21,763 @@
-      a0[i] = (int)(a1[i]+a2[i]);
-    }
-  }
-
-  static void test_subc(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]-VALUE);
-    }
-  }
-  static void test_subv(int[] a0, int[] a1, int b) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]-b);
-    }
-  }
-  static void test_suba(int[] a0, int[] a1, int[] a2) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]-a2[i]);
-    }
-  }
-
-  static void test_mulc(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]*VALUE);
-    }
-  }
-  static void test_mulc_n(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]*(-VALUE));
-    }
-  }
-  static void test_mulv(int[] a0, int[] a1, int b) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]*b);
-    }
-  }
-  static void test_mula(int[] a0, int[] a1, int[] a2) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]*a2[i]);
-    }
-  }
-
-  static void test_divc(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]/VALUE);
-    }
-  }
-  static void test_divc_n(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]/(-VALUE));
-    }
-  }
-  static void test_divv(int[] a0, int[] a1, int b) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]/b);
-    }
-  }
-  static void test_diva(int[] a0, int[] a1, int[] a2) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]/a2[i]);
-    }
-  }
-
-  static void test_andc(int[] a0, int[] a1) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]&BIT_MASK);
-    }
-  }
-  static void test_andv(int[] a0, int[] a1, int b) {
-    for (int i = 0; i < a0.length; i+=1) {
-      a0[i] = (int)(a1[i]&b);
+
+    private static final int ARRLEN = 997;
+    private static final int ITERS  = 11000;
+    private static final int ADD_INIT = Integer.MAX_VALUE-500;
+    private static final int BIT_MASK = 0xEC80F731;
+    private static final int VALUE = 15;
+    private static final int SHIFT = 32;
+
+    public static void main(String[] args) {
+        TestFramework.runWithFlags("-XX:+IgnoreUnrecognizedVMOptions",
+                                   "-XX:StressLongCountedLoop=0");
+    }
+
+    @Run(test = {
+           "test_addc", "test_addv", "test_adda", "test_subc", "test_subv",
+           "test_suba", "test_mulc", "test_mulc_n", "test_mulv", "test_mula",
+           "test_andc", "test_andv", "test_anda", "test_orc", "test_orv",
+           "test_ora", "test_xorc", "test_xorv", "test_xora", "test_sllc",
+           "test_sllc_n", "test_sllv", "test_srlc", "test_srlc_n", "test_srlv",
+           "test_srac", "test_srac_n", "test_srav"
+         },
+         mode = RunMode.STANDALONE)
+    public void run() {
+
+        System.out.println("Testing Integer vectors");
```

Comment:
Not sure if it's worth to keep these printing statements. But does not hurt either to leave them in.